### DISCUSSION TITLE
What is "sentry.wizarr.dev"? And what data is this collecting from my server?

### DISCUSSION THREAD

**Discussion by trentnbauer** [wizarrrr/wizarr#401] [Q&A] [answered]
Hi all,
I've noticed the below in my logs;
![image](https://github.com/wizarrrr/wizarr/assets/45578026/a2a72252-68b0-4122-ba40-012da227fcd3)

Navigating to the URL gives a Github login;
![image](https://github.com/wizarrrr/wizarr/assets/45578026/fa7b603a-0fba-4888-98d2-c0014e3f701b)
Which I do not have access too;
![image](https://github.com/wizarrrr/wizarr/assets/45578026/8bfb361f-e399-4716-959b-08d15a68ed46)

What is Wizarr Sentry? What data is this grabbing from my instance?
I can't find any reference to it online...

**Comment by mrdynamo** [comment, marked as answer]
Hi @trentnbauer

Sentry is an industry standard application to monitor performance and error reporting within an application. It sends general information about the environment and stack traces for any errors caught within your instance. 

This allows developers to easily see where the error occurred and information that can help us fix bugs or general issues.

There is no personal information shared to us. Anything personal (API keys, etc) is redacted prior to being sent to the Sentry instance.

If you wish, you may disable this feature by disabling bug reporting within Wizarr settings.

If you have further questions about Sentry, don't hesitate to ask!

More information can be found on [Sentry's website](https://sentry.io/about/).
